### PR TITLE
sync with develop (20241016)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Review by branfosj — APPROVED** [review]

File: easybuild/easyconfigs/b/BamTools/BamTools-2.5.2-GCC-13.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
```
Comment (branfosj) [inline easybuild/easyconfigs/b/BamTools/BamTools-2.5.2-GCC-13.2.0.eb:1]:
```suggestion
# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
easyblock = 'CMakeMake'

```

File: easybuild/easyconfigs/b/BamTools/BamTools-2.5.2-GCC-13.2.0.eb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+name = 'BamTools'
+version = '2.5.2'
+
+homepage = 'https://github.com/pezmaster31/bamtools'
+description = "BamTools provides both a programmer's API and an end-user's toolkit for handling BAM files."
+
+toolchain = {'name': 'GCC', 'version': '13.2.0'}
+toolchainopts = {'pic': True}
+
```
Comment (branfosj) [inline easybuild/easyconfigs/b/BamTools/BamTools-2.5.2-GCC-13.2.0.eb:10]:
```suggestion

# https://github.com/pezmaster31/bamtools
github_account = 'pezmaster31'
```

File: easybuild/easyconfigs/b/BamTools/BamTools-2.5.2-GCC-13.2.0.eb [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+sources = ['v%(version)s.tar.gz']
+checksums = ['4d8b84bd07b673d0ed41031348f10ca98dd6fa6a4460f9b9668d6f1d4084dfc8']
+
+builddependencies = [
+    ('CMake', '3.27.6'),
+]
+
+# https://github.com/pezmaster31/bamtools
+github_account = 'pezmaster31'
+
```
Comment (branfosj) [inline easybuild/easyconfigs/b/BamTools/BamTools-2.5.2-GCC-13.2.0.eb:21]:
```suggestion
sanity_check_paths = {
    'files': ['bin/bamtools', 'include/bamtools/shared/bamtools_global.h', 'lib/libbamtools.a'],
    'dirs': ['include/bamtools/api', 'lib/pkgconfig'],
}

sanity_check_commands = ["bamtools --help"]

```

**Review by branfosj — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/p/popt/popt-1.18.eb [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+homepage = "http://freecode.com/projects/popt"
+description = """Popt is a C library for parsing command line parameters."""
+toolchain = SYSTEM
+
+source_urls = ['http://ftp.rpm.org/popt/releases/popt-1.x/']
+sources = [SOURCE_TAR_GZ]
+checksums = ['5159bc03a20b28ce363aa96765f37df99ea4d8850b1ece17d1e6ad5c24fdc5d1']
+
+toolchainopts = {'optarch': False}
```
Comment (branfosj) [inline easybuild/easyconfigs/p/popt/popt-1.18.eb:14]:
```suggestion
```

**Review by branfosj — APPROVED** [review]

File: easybuild/easyconfigs/p/popt/popt-1.18.eb (hunk 4 above)
Comment (boegel) [reply easybuild/easyconfigs/p/popt/popt-1.18.eb]: fixed in 3685424

**Pull request merged**